flink-runtime-web/web-dashboard/app/scripts/modules/jobs/jobs.dir.coffee

```diff
@@ -281,7 +281,8 @@ angular.module('flinkApp')
         # Otherwise add infos    
         labelValue += "<h5>" + info + " Node</h5>"  if isSpecialIterationNode(info)
         labelValue += "<h5>Parallelism: " + el.parallelism + "</h5>"  unless el.parallelism is ""
-        labelValue += "<h5>Operation: " + shortenString(el.operator_strategy) + "</h5>"  unless el.operator is `undefined`
+        if el.operator_strategy
+          labelValue += "<h5>Operation: " + shortenString(el.operator_strategy) + "</h5>"  unless el.operator is `undefined`
```

Comment:
It's fine, but maybe:
```
labelValue += "<h5>Operation: " + shortenString(el.operator_strategy) + "</h5>"  unless !el.operator_strategy || el.operator is `undefined`
```

doesn't really matter much, can be as well as you've put it.